_observing-your-data/ad/index.md

```diff
@@ -32,88 +32,113 @@ A _detector_ is an individual anomaly detection task. You can define multiple de
-
-8. Choose **Next**.   
+
+    ```json
+     {
+        "bool": {
+           "should": [
+                 {
+                    "term": {
+                       "urlPath.keyword": "/domain/{id}/short"
+                    }
+                 },
+                 {
+                    "term": {
+                       "urlPath.keyword": "/sub_dir/{id}/short"
+                    }
+                 },
+                 {
+                    "term": {
+                       "urlPath.keyword": "/abcd/123/{id}/xyz"
+                    }
+                 }
+           ]
+        }
+     }
+     ```
+     {% include copy-curl.html %}
+
+#### Setting the detector interval   
+
+In the **Timestamp** pane, select a field from the **Timestamp field** dropdown menu. 
+
+Then, in the **Operation settings** pane, use the following best practices to define the **Detector interval**, which is the interval at which the detector collects data.
+
+- The detector aggregates the data at this interval and then feeds the aggregated result into the anomaly detection model. The shorter the interval, the fewer data points the detector aggregates. The anomaly detection model uses a shingling process, a technique that uses consecutive data points to create a sample for the model. This process requires a certain number of aggregated data points from contiguous intervals.
+- You should set the detector interval based on your actual data. If the detector interval is too long, then it might delay the results. If the detector interval is too short, then it might miss some data. The detector interval also will not have a sufficient number of consecutive data points for the shingle process.
+- (Optional) To add extra processing time for data collection, specify a **Window delay** value.
+  - This value tells the detector that the data is not ingested into OpenSearch in real time but with a certain delay. Set the window delay to shift the detector interval to account for this delay.
+  - For example, the detector interval is 10 minutes and data is ingested into your cluster with a general delay of 1 minute. Assume the detector runs at 2:00. The detector attempts to get the last 10 minutes of data from 1:50 to 2:00, but because of the 1-minute delay, it only gets 9 minutes of data and misses the data from 1:59 to 2:00. Setting the window delay to 1 minute shifts the interval window to 1:49--1:59, so the detector accounts for all 10 minutes of the detector interval time.
+  - To avoid missing any data, set the **Window delay** to the upper limit of the expected ingestion delay. This ensures that the detector captures all data during its interval, reducing the risk of missing relevant information. While a longer window delay helps capture all data, too long of a window delay can hinder real-time anomaly detection because the detector will look further back in time. Find a balance to maintain both data accuracy and timely detection.
```

Comment:
```suggestion
  - To avoid missing any data, set the **Window delay** to the upper limit of the expected ingestion delay. This ensures that the detector captures all data during its interval, reducing the risk of missing relevant information. While a longer window delay helps capture all data, too long of a window delay can hinder real-time anomaly detection because the detector will look further back in time. Find a balance that maintains both data accuracy and timely detection.
```